tests/test_core.py

```diff
@@ -5068,6 +5068,9 @@ def test_libgen(self):
   def test_utime(self):
     self.do_runf(path_from_root('tests', 'utime', 'test_utime.c'), 'success')
 
+  def test_futimens(self):
+    self.do_runf(path_from_root('tests', 'utime', 'test_futimens.c'), 'success')
```

Comment:
As this test is for JS library support, and not code generation, please move it to `other`. That way it will not run in all different optimization modes unnecessarily.

To do that, see e.g. `test_unlink` in `tests/test_other.py`. Basically you emit something like
```
  def test_unlink(self):
    self.do_other_test('test_unlink.cpp')
```
and the test files go in `tests/other/`.

Reply:
Once this is done I think we can go ahead and land this.

Reply:
Actually I don't see how this test is any different to the one directly above it?  Wouldn't it be better to keep them together?  (we can consider moving them to other as a followup perhaps?)

Reply:
Been on vacation.  

So what do we do here???  It seems like the only thing holding this up is where the test lives and how??

1) A test was created from the start.
2) Test was asked to be separate out which was done.
3) Looks like that was not right and test should be done a different way?
4) Now it is another way and "(we can consider moving them to other as a followup perhaps?)".

It is a small test but the fix is needed to allow us to upgrade to the recent as it is a breaking regression.  Is there any way to get this in and once the test is decided on where it should live and how then it can be done?  

If the test is no different than the one directly above it then it should have caught this problem unless I misunderstand the question.

Reply:
That's fair @kjpou1 , let's land this and figure out the testing details later.